autoload/neomake/makers/ft/verilog.vim

```diff
@@ -4,7 +4,7 @@ endfunction
 
 function! neomake#makers#ft#verilog#iverilog() abort
     return {
-                \ 'args' : ['-t null', '-Wall'],
+                \ 'args' : ['-tnull', '-Wall', '-y./'],
```

Comment:
What does `-y./` do exactly?
Shouldn't it be rather be `%:h`, i.e. the directory of the file?
Or even a project root?

Reply:
It includes the dir where other modules are defined. The use case is being able to compile a file even if a module is defined in a separate file.
Honestly, I "copied" it from the C maker that includes the current dir with `-I./`.
If you want I can replace it with `-y%:h`.